scripts/add-toggle.sh

```diff
@@ -9,8 +9,12 @@ export JSON_DIR="/scripts/"
 export TOGGLE_NAME="${1}"
 export DESCRIPTION="${2}"
 export TOGGLE_ENABLED="${3}"
-export ADMIN_USER="${4}"
-export ADMIN_PW="${5}"
+
+if [ "_${IMPORTER_CREDS_MOUNT}" != "_" ]; then
+  echo "Getting credentials from vault"
+  ADMIN_USERNAME=$(cat ${IMPORTER_CREDS_MOUNT}/admin-username-cmc)
```

Comment:
i will get working with this - then think about creating a generic secrets for user/pw: `admin-username-for-pr` and `admin-password-for-pr`